runbot_pylint/models/runbot_build.py

```diff
@@ -91,8 +91,8 @@ def get_repo_branch_name(self):
         for build in self:
             hint_branches = set()
             for extra_repo in build.repo_id.dependency_ids:
-                closest_name = build.get_closest_branch_name(
-                    extra_repo.id, hint_branches)
+                repo_id, closest_name = build._get_closest_branch_name(
+                    extra_repo.id)
```

Comment:
If you're not using repo_id, make `closest_name = build._get_closest_branch_name(extra_repo.id)[1]`